src/Appium.Net/Appium/Windows/WindowsOptions.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using static System.String;
+
+namespace OpenQA.Selenium.Appium.Windows
+{
+    public class WindowsOptions : AppiumOptions
+    {
+        private new string VendorPrefix = "ms";
+        private const string PlatformNameOption = "platformName";
+        private new const string AutomationNameOption = "automationName";
+        private new const string AppOption = "app";
+        private const string AppArgumentsOption = "appArguments";
+        private const string AppTopLevelWindowOption = "appTopLevelWindow";
+        private const string AppWorkingDirOption = "appWorkingDir";
+        private const string CreateSessionTimeoutOption = "createSessionTimeout";
+        private const string WaitForAppLaunchOption = "ms:waitForAppLaunch";
```

Comment:
I mean that I can observe some unnecessary duplication where we define a separate constant for `ms` above and then still use the string itself in other constants instead of using the variable.

Reply:
Oh I got rid of that constant.